output/csv/output.go

```diff
@@ -200,9 +204,17 @@ func MakeHeader(tags []string) []string {
 }
 
 // SampleToRow converts sample into array of strings
-func SampleToRow(sample *metrics.Sample, resTags []string, ignoredTags []string, row []string) []string {
+func SampleToRow(sample *metrics.Sample, resTags []string, ignoredTags []string, row []string,
+	timeFormat TimeFormat,
+) []string {
 	row[0] = sample.Metric.Name
-	row[1] = fmt.Sprintf("%d", sample.Time.Unix())
+
+	if timeFormat == RFC3399 {
+		row[1] = fmt.Sprint(sample.Time.Format(time.RFC3339))
```

Comment:
```suggestion
		row[1] = sample.Time.Format(time.RFC3339)
```
Format returns already a string

Reply:
Thanks, will make this change.